### PR TITLE
py-reportlab: add missing dependency on freetype

### PR DESCRIPTION
**Pull request by glennpj**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-reportlab/package.py [hunk 1]
```diff
@@ -19,3 +19,5 @@ class PyReportlab(PythonPackage):
     # - easy_install, provided by py-setuptools
     # - pip, provided by py-pip
     extends('python', ignore=r'bin/.*')
+
+    depends_on('freetype')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-reportlab/package.py:26]: There's also a bunch of other missing deps:
```python
install_requires=['pillow>=2.4.0','pip>=1.4.1', 'setuptools>=2.2'],
```
Also, it looks like there are a few env vars you can set to help the build find freetype if you notice any linking issues.

File: var/spack/repos/builtin/packages/py-reportlab/package.py [hunk 2]
```diff
@@ -19,3 +19,8 @@ class PyReportlab(PythonPackage):
     # - easy_install, provided by py-setuptools
     # - pip, provided by py-pip
     extends('python', ignore=r'bin/.*')
+
+    depends_on('py-setuptools@2.2:', type='build')
+    depends_on('py-pip@1.4.1:', type='build')
+    depends_on('py-pillow@2.4.0:', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-reportlab/package.py]:
```suggestion
    depends_on('pil@2.4.0:', type=('build', 'run'))
```
We have a virtual dep for this

**Review by adamjstewart — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/py-reportlab/package.py (hunk 2 above)
Comment (glennpj) [reply var/spack/repos/builtin/packages/py-reportlab/package.py]: I was not aware of that. Will the version stay the same? It looks like the majority of the pil providers are different versions of py-pillow.

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-reportlab/package.py]: Yes, versions are compatible. I always use py-pillow-simd since it's much faster.

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**